tests/start_network.py

```diff
@@ -113,6 +113,16 @@ def pad_node_id(nid):
                     )
                 )
 
+            if args.auto_shutdown:
+                if args.auto_shutdown_delay_s > 0:
+                    LOG.info(
+                        f"Waiting {args.auto_shutdown_delay_s}s before automatic shutdown..."
+                    )
+                    time.sleep(args.auto_shutdown_delay_s)
+
+                LOG.info("Automatically shut down network after successful opening")
+                return
```

Comment:
Why here rather than at line 140? If anyone ever runs this by hand, it would be nice to keep the logging similar (and still allow `Ctrl+C` to terminate the service earlier than the auto shutdown?)

Reply:
Yes you're right. I started with the no delay case first, but this doesn't make sense with the delayed shutdown case. Fixed now.